include/zephyr/net/phy.h

```diff
@@ -83,6 +83,55 @@ struct phy_link_state {
 	bool is_up;
 };
 
+struct phy_plca_cfg {
```

Comment:
Maybe a comment block with description of fields would help?

Reply:
Ah good point. Will add comments.